### PR TITLE
added beerpay.io itegration

### PR DESCRIPTION
**Pull request by colkito**
This PR adds support for beerpay.io, as requested in #659.

Fixes: #659 

Thanks for considering 👍
@colkito

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -4711,6 +4711,35 @@ cache(function(data, match, sendBadge, request) {
   })}
 ));
 
+// beerpay.io integration.
+camp.route(/^\/beerpay\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1],
+      project = match[2],
```
Comment (espadrine) [inline server.js]: Could you add example values in comments, please?

File: server.js [hunk 2]
```diff
@@ -4711,6 +4711,35 @@ cache(function(data, match, sendBadge, request) {
...
+      format = match[3];
```
Comment (espadrine) [inline server.js]: Could you put each line as a var statement, please?

**Comment by espadrine** [comment]
Thanks for the contribution! Could you add an example badge on try.html please?

**Comment by colkito** [comment]
Done @espadrine!
**[OT]** did you think about create a modular structure in order to organize the main server.js code? I could work on that if you agree.
e.g. /integrations/beerpay.js, /integrations/travis.js, etc.

Thanks!

**Comment by espadrine** [comment]
Thanks for your contribution! It's live. I'll update the website in 24h.

Out of curiosity, what are the pros and cons of separating server.js into one file for each integration? You can definitely create a new issue to pursue the discussion!

**Comment by colkito** [comment]
IMO: using a single file for each integration will be easiest for new contributors. Clean code and clean examples. Will be a great idea have a example.js file to edit for new contributions.

If you are agree, I could create the issue to discussion.

best

**Pull request merged**